### ISSUE TITLE
Updating Deprecated/Removed Java calls.

### ISSUE THREAD

**Issue by saulhidalgoaular** [debrief/debrief#5173]
The import org.eclipse.core.databinding.beans.BeansObservables cannot be resolved
The import org.eclipse.jface.databinding.swt.WidgetProperties cannot be resolved

Calls to old Java databindings methods are causing are compilation error in `org.mwc.debrief.satc.core`

Bundle 'org.eclipse.wst.xml.ui' cannot be resolved

Missing library in `org.mwc.asset.core`

**Comment by IanMayo** [comment]
Yes please @saulhidalgoaular - well spotted.

**Comment by saulhidalgoaular** [comment]
I had this compilation error, I googled the reasons, and I remember I worked on solving it, but somehow I don't find the changes I did in the Capture Plot to file feature PR.

In any case, I have tested it and now It compiles and runs on Java 11.0.17 on Ubuntu 20.04.

![image](https://user-images.githubusercontent.com/11049943/235587624-1ea52893-db26-4658-85d2-07365838c4ba.png)

It also compiles on the PR.

![image](https://user-images.githubusercontent.com/11049943/235587864-2e132912-8420-4f43-aa10-06a7f36c40e3.png)

So I consider safe to mask this issue.
